### DISCUSSION TITLE
Mixbox for ToonSquid

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#277] [Ideas]
You guys know Secret Weapons, the guys who made EbSynth, the tool people are using to create living paintings with video and a single edited frame? Well, they have this thing called Mixbox, where you can mix two different colors like you can in real life. If ToonSquid had this, it could add even more to what's already the perfect iPad animation app. I never shared a YouTube video on GitHub before, so here's my first attempt.  [Natural Color Mixing with Mixbox](https://youtu.be/F2_JQG6YcQ4)

**Comment by Mokaman51** [comment]
ArtStudio Pro introduced something like this recently and it's great. I'd also love to see this in ToonSquid
